src/transaction.spec.ts

```diff
@@ -614,4 +614,44 @@ describe("test transaction", async () => {
         transaction.applyGuardianSignature(transaction.getSignature());
         assert.isTrue(transaction.isGuardedTransaction());
     });
+
+    it("test sign using hash", async () => {
+        let transaction = new Transaction({
+            nonce: 89n,
+            value: 0n,
+            sender: wallets.alice.address.toBech32(),
+            receiver: wallets.bob.address.toBech32(),
+            gasLimit: 50000n,
+            gasPrice: 1000000000n,
+            chainID: "integration tests chain ID",
+            version: 2,
+            options: 1,
```

Comment:
In the specs, we should add an extra method on the computer - e.g. _apply option sign using hash._

Reply:
Created a task, will do.